include/crocoddyl/core/activations/quadratic-barrier.hpp

```diff
@@ -48,10 +48,10 @@ struct ActivationBoundsTpl {
         }
       }
       // Assign the maximum value for infinity/nan values
-      if (!std::isfinite(lb(i))) {
```

Comment:
You should use `using namespace std`.

Reply:
In any case, this code is not compatible with codegen because of the if statements !!!

Reply:
I think just keeping the `using std::isfinite` (https://github.com/loco-3d/crocoddyl/pull/899#discussion_r605476858) is enough here.
I agree about the if statements, they need to be handled separately. For now, I don't' think this maximum value setting would work in codegen.

Reply:
Are you sure this code won't work for codegen?

I am asking it as this if-statement is in the class constructor.

(I have solved this issue by adding `using std::isfinite`)

Reply:
It depends if you follow or not the design pattern of Pinocchio with the cast methods.

Reply:
I am not aware of these details. @proyan -- could you let us know?

(I will mark this conversation as resolved)